tests/e2e/models/CMakeLists.txt

```diff
@@ -85,3 +85,79 @@ iree_check_single_backend_test_suite(
 )
 
 ### BAZEL_TO_CMAKE_PRESERVES_ALL_CONTENT_BELOW_THIS_LINE ###
+
+iree_static_linker_test(
+  NAME
+    mnist_fake_weights_llvm_cpu_static_bytecode_test
+  SRC
+    "mnist_fake_weights.mlir"
+  STATIC_LIB_PREFIX
+    mnist_fake_weights_linked_llvm_cpu
+  MAIN_FUNCTION
+    "module.predict"
+  INPUT_SHAPES
+    "1, 28, 28, 1"
+  INPUT_TYPE
+    float
+  HAL_TYPE
+    IREE_HAL_ELEMENT_TYPE_FLOAT_32
+  COMPILER_FLAGS
+    "--iree-input-type=mhlo"
+)
+
+iree_static_linker_test(
+  NAME
+    mnist_fake_weights_llvm_cpu_static_c_test
+  SRC
+    "mnist_fake_weights.mlir"
+  STATIC_LIB_PREFIX
+    mnist_fake_weights_linked_llvm_cpu
+  MAIN_FUNCTION
+    "module.predict"
+  INPUT_SHAPES
+    "1, 28, 28, 1"
+  INPUT_TYPE
+    float
+  HAL_TYPE
+    IREE_HAL_ELEMENT_TYPE_FLOAT_32
+  COMPILER_FLAGS
+    "--iree-input-type=mhlo"
+  EMITC
+)
+
+iree_static_linker_test(
```

Comment:
These two tests are out of place. I can't find an example in `tests` that has multiple inputs. I can also make a copy of the `simple_mul.mlir` there to make it more relevant to the directory.

Reply:
Related to my other comment, using the standard `"1x384xi32,1x384xi32,1x384xi32"` style would be nice here https://github.com/iree-org/iree/blob/e33c64c8c1b58958cf56982372e53332c5c1aeb2/benchmarks/TFLite/CMakeLists.txt#L69-L72

Reply:
Done.